File: Modul-2/Soal 5/PRAK205-2410817210011-MUHAMMADDZULFATHIAHYAN.c
#include <stdio.h>
#include <math.h>
int main() 
{
    float tihang,singgang;
    printf("Masukkan nilai tinggi dan miring\n");
    scanf("%f",&tihang);
    scanf("%f",&singgang);
    //40 41 dan 16 65
    printf("Alas = %.0f cm\n",sqrt((singgang*singgang)-(tihang*tihang)));
    printf("Tinggi = %.0f cm\n",tihang);
    printf("Keliling = %.0f cm\n",tihang+(sqrt((singgang*singgang)-(tihang*tihang)))+singgang);
    printf("Luas = %.0f cm^2\n",0.5*(sqrt((singgang*singgang)-(tihang*tihang)))*tihang);

    return 0;
}

File: Modul-2/Soal 5/PRAK205-2410817210011-MUHAMMADDZULFATHIAHYAN.py
nilai=[]
while (len(nilai)<2):
    baris=input()
    baris1=baris.split()

    for value in baris1:
        nilai.append(float(value))
tihang,singgang=nilai
#40 41 dan 16 65
print(f"Alas = {((singgang*singgang)-(tihang*tihang))**0.5:.0f} cm")
print(f"Tinggi = {tihang:.0f} cm")
print(f"Keliling = {tihang+(((singgang*singgang)-(tihang*tihang))**0.5)+singgang:.0f} cm")
print(f"Luas = {0.5*(((singgang*singgang)-(tihang*tihang))**0.5)*tihang:.0f} cm^2")